### PR TITLE
Use SpringBoot for REST server (#111)

### PR DESCRIPTION
**Pull request by itskarlsson**
Closes issue #111

### CODE REVIEW

File: rest/src/main/java/com/ericsson/bss/cassandra/ecchronos/rest/RepairManagementRESTImpl.java [hunk 1]
```diff
@@ -34,26 +38,38 @@
...
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
 
 /**
  * When updating the path it should also be updated in the OSGi component.
  */
-@Path("/repair-management/v1")
+@RestController
+@Configuration
+@ComponentScan
```
Comment (emolsson) [inline rest/src/main/java/com/ericsson/bss/cassandra/ecchronos/rest/RepairManagementRESTImpl.java:51]: The @ComponentScan/@Configuration annotation does not seem to do anything. The python-tests seems to work without them at least.

File: pom.xml [hunk 2]
```diff
@@ -203,47 +201,31 @@
...
-                <version>${jersey.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.glassfish.jersey.containers</groupId>
-                <artifactId>jersey-container-servlet-core</artifactId>
-                <version>${jersey.version}</version>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-test</artifactId>
+                <version>${org.springframework.boot.version}</version>
```
[the remark this answers is not in the document]
Comment (itskarlsson) [reply pom.xml:221]: We use this in rest for testing purposes, added test scope

[the remark this answers is not in the document]
Comment (itskarlsson) [reply pom.xml:221]: Actually, you are right. I solved it in a different way that doesn't use that package.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java [hunk 3]
```diff
@@ -60,6 +62,13 @@ private static Config getConfiguration() throws ConfigurationException
         }
     }
 
+    @Bean
+    public ConfigurableServletWebServerFactory webServerFactory(Config configuration) {
+        TomcatServletWebServerFactory factory = new TomcatServletWebServerFactory();
+        factory.setPort(configuration.getRestServer().getPort());
```
Comment (emolsson) [inline application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java:68]: We seem to ignore the host config still

**Review by emolsson — APPROVED** [review]

**Pull request merged**